### PR TITLE
Add cookie sameSite settings

### PR DESCRIPTION
**Pull request by zengyan-amazon**
*Issue #, if available:*

*Description of changes:* Read and set `sameSite` option for cookie from config

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yuxitang-amzn — APPROVED** [review]

File: server/session/security_cookie.ts [hunk 1]
```diff
@@ -69,5 +69,6 @@ export function getSecurityCookieOptions(
       return { isValid: true, path: '/' };
     },
     isSecure: config.cookie.secure,
+    sameSite: config.cookie.isSameSite || 'None',
```
Comment (yuxitang-amzn) [inline server/session/security_cookie.ts:72]: So if `isSameSite` is `false`, it will become `'None'` here, is it expected?

File: server/index.ts [hunk 2]
```diff
@@ -42,17 +42,8 @@ export const configSchema = schema.object({
...
-          validate(value) {
-            if (value === 'Strict' || value === 'Lax') {
-              return `Allowed values of 'isSameSite' are ['Strict, 'Lax', true, false]`;
-            }
-          },
-        }),
-        schema.boolean(),
-      ],
-      { defaultValue: true }
+      [schema.literal('Strict'), schema.literal('Lax'), schema.literal(false)],
```
Comment (tianleh) [inline server/index.ts:45]: Just want to confirm that "true" is intentionally excluded.

File: server/session/security_cookie.ts (hunk 1 above)
Comment (zengyan-amazon) [reply server/session/security_cookie.ts:72]: yes expected, basically Kibana accepts `Strict`, `Lax` and `None`, while `false` is meant to be `None` in `kibana.yml` config to make it parity with the legacy config settings

File: server/index.ts (hunk 2 above)
Comment (zengyan-amazon) [reply server/index.ts:45]: yes, same reason as my comment above

**Pull request merged**